core/variant_parser.cpp

```diff
@@ -1099,7 +1099,7 @@ Error VariantParser::parse_value(Token &token, Variant &value, Stream *p_stream,
 
 			return OK;
 
-		} else if (id == "PoolFloatArray" || id == "FloatArray") {
+		} else if (id == "PoolRealArray" || id == "RealArray") {
```

Comment:
Yeah, I'm not really sure about this change. Besides, it doesn't really pertain to the issue and deserves another pull request. I have removed the changes to this file.